WSL/wsl-config.md

```diff
@@ -144,6 +144,23 @@ wsl.conf section label: `[boot]`
 | Key | Value | Default | Notes |
 |:----|:----|:----|:----|
 | `command` | string | `""` | A string of the command that you would like to run when the WSL instance starts. This command is run as the root user. e.g: `service docker start`. |
+| `protectBinfmt` | boolean | `true` | Prevents WSL from generating systemd units when systemd is enabled  |
+
+### GPU settings
+
+wsl.conf section label: `[gpu]`
+
+| Key | Value | Default | Notes |
+|:----|:----|:----|:----|
+| `enabled` | boolean | `true` | Allow Linux applications to access the Windows GPU via para-virtualization. |
```

Comment:
does the default value of this is true?

Reply:
Yes true is the default

Reply:
so, is this is the current behavior with mesa and d3d12?

Reply:
Yes!